model/commitqueue/db.go

```diff
@@ -53,13 +54,13 @@ func insert(q *CommitQueue) error {
 func add(id string, queue []CommitQueueItem, item CommitQueueItem) error {
 	err := updateOne(
 		bson.M{
-			IdKey:    id,
-			QueueKey: queue,
+			IdKey: id,
 		},
 		bson.M{"$push": bson.M{QueueKey: item}},
 	)
 
 	if adb.ResultsNotFound(err) {
+		grip.Error(errors.Wrapf(err, "update failed for queue '%s', %+v", id, queue))
 		return errors.New("queue has changed in the database")
```

Comment:
This message is no longer correct. Return the same thing you log?

Reply:
done.